app/models/developer.rb

```diff
@@ -27,13 +24,16 @@ class Developer < ApplicationRecord
   has_many :conversations, -> { visible }
   has_many :messages, -> { where(sender_type: Developer.name) }, through: :conversations
   has_many :hired_forms, class_name: "Hired::Form", dependent: :destroy
+  has_one :badge, dependent: :destroy, class_name: "Developers::Badge"
   has_one :location, dependent: :destroy, autosave: true
   has_one :role_level, dependent: :destroy, autosave: true
   has_one :role_type, dependent: :destroy, autosave: true
   has_one_attached :cover_image
 
   has_noticed_notifications
 
+  delegate :source_contributor?, :recently_active?, to: :badge
```

Comment:
What's the purpose of this? We are already delegating in the `Developers::BadgeComponent`.